frontend/src/Components/AddressLookupComponent/AddressLookupComponent.jsx

```diff
@@ -1,44 +1 @@
-// import React, { useState, useEffect } from "react";
```

Comment:
@SemanBoon Could you share why we're deleting all the code in this file? Also, reminder to have commit messages reflect the changes that are happening within it :)